tools/benchmark/cmd/mvcc-put.go

```diff
@@ -106,16 +108,25 @@ func mvccPutFunc(cmd *cobra.Command, args []string) {
 	rrc := r.Results()
 
 	rc := r.Run()
-	for i := 0; i < totalNrKeys; i++ {
-		st := time.Now()
-		if txn {
+
+	if txn {
+		for i := 0; i < totalNrKeys; i += nrBatch {
+			st := time.Now()
+
 			tw := s.Write()
-			tw.Put(keys[i], vals[i], lease.NoLease)
+			for j := i; j < i+nrBatch; j++ {
+				tw.Put(keys[j], vals[j], lease.NoLease)
+			}
 			tw.End()
-		} else {
+
+			rrc <- report.Result{Start: st, End: time.Now(), Ops: uint(nrBatch)}
```

Comment:
I feel like the `Ops` field unnecessarily complicates the report code. If the benchmark is oriented around requests (instead of keys written) I don't think there's a need to have the multi-op counting